src/components/ChannelList/hooks/listeners/useChannelDeleted.js

```diff
@@ -0,0 +1,28 @@
+import { useContext, useEffect } from 'react';
+
+import { ChatContext } from '../../../../context';
+
+export const useChannelDeleted = ({ onChannelDeleted, setChannels }) => {
+  const { client } = useContext(ChatContext);
+
+  useEffect(() => {
+    const handleEvent = (e) => {
+      if (onChannelDeleted && typeof onChannelDeleted === 'function') {
```

Comment:
made this comment in few places
```suggestion
      if (typeof onChannelDeleted === 'function') {
```